src/workload.rs

```diff
@@ -132,10 +156,11 @@ impl Workload {
         }
     }
     pub fn choose_waypoint_address(&self) -> Option<IpAddr> {
```

Comment:
name is a bit misleading now since its not actually _choosing_ anymore.

Reply:
is it to return the waypoint cluster ip?

Reply:
see 9ec9a1260fce16f8abc343596431ccb2fdf9a39f